examples/panel_stream.html

```diff
@@ -52,7 +52,7 @@
 	  <a class="navbar-brand app-logo" href="/">
 	    <img src="./logo.png" class="app-logo">
 	  </a>
-	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>
+	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>**
```

Comment:
Why this change?

Reply:
oh I think I added this by the mistake
```suggestion
	  <a class="title" href="" style="color: #f0ab3c;">Panel Streaming Demo</a>
```